src/components/EventDashboard/CCPTabPanel.tsx

```diff
@@ -108,16 +110,34 @@ const CCPTabPanel = ({ eventId }: { eventId: string }) => {
   const [orderBy, setOrderBy] = React.useState<string>('name');
   const classes = useStyles();
   const [order, setOrder] = React.useState<Order>('asc');
+  const [anchorEl, setAnchorEl] = React.useState<
+    null | (EventTarget & HTMLButtonElement)
+  >(null);
   const { data } = useQuery(GET_CCPS_BY_EVENT_ID, { variables: { eventId } });
 
   const rows = data ? data.collectionPointsByEvent : [];
+  const open = Boolean(anchorEl);
+  const history = useHistory();
+  const id = open ? 'simple-popper' : undefined;
 
   const handleRequestSort = (event: React.MouseEvent, property: string) => {
     const isAsc = orderBy === property && order === 'asc';
     setOrder(isAsc ? 'desc' : 'asc');
     setOrderBy(property);
   };
 
+  const handleAdd = () => {
+    history.replace(`/events/${eventId}/new`);
+  }
+
+  const handleEdit = () => {
+    history.replace(`/events/${eventId}/ccps/`);
+  }
+
+  const handleDelete = () => {
+
```

Comment:
let's add something here so it doesn't look out of place
```suggestion
// TODO: handle delete here
```

Reply:
Added.